.cirrus.yml

```diff
@@ -37,6 +37,8 @@ task:
     # -----
     cat /proc/cpuinfo
   install_libvirt_vagrant_script: |
+    wget -O- https://apt.releases.hashicorp.com/gpg | sudo apt-key add -
```

Comment:
note that `apt-key` is deprecated (also see https://github.com/docker/docs/pull/11990), so would be something like;

```bash
 wget-O- https://apt.releases.hashicorp.com/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/hashicorp.gpg
```

or (making sure the directory exists, and set permissions; see https://github.com/docker/docs/pull/17070)

```bash
 sudo install -m 0755 -d /etc/apt/keyrings
 wget-O- https://apt.releases.hashicorp.com/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/hashicorp.gpg
 sudo chmod a+r /etc/apt/keyrings/hashicorp.gpg
```

Reply:
Thanks; done.